docs/x509/reference.rst

```diff
@@ -463,6 +463,15 @@ X.509 CRL (Certificate Revocation List) Object
             >>> crl.fingerprint(hashes.SHA256())
             b'e\xcf.\xc4:\x83?1\xdc\xf3\xfc\x95\xd7\xb3\x87\xb3\x8e\xf8\xb93!\x87\x07\x9d\x1b\xb4!\xb9\xe4W\xf4\x1f'
 
+    .. method:: get_revoked_certificate(serial_number)
```

Comment:
I think this should be `get_revoked_certificate_by_serial_number`?

Reply:
Yeah that's more consistent with `get_values_for_type`, `get_extension_for_oid`, `get_extension_for_class`. Changing.